lib/form/paragraph3.flow

```diff
@@ -44,6 +44,7 @@ reconstructed on each reflow, because we combine them together to have as few sp
 possible, and because rendering words in the same sprite improves the interword spacing.
 */
 
+LARGE = 100000.0;
```

Comment:
bad naming. the name should describe the purposes of the variable/constant.
for example, paragraphWidthLarge looks better candidate

Reply:
Ok.